hathor/merged_mining/coordinator.py

```diff
@@ -259,7 +373,7 @@ class MergedMiningStratumProtocol(asyncio.Protocol):
     DEFAULT_XNONCE2_SIZE = 8  # size in bytes to reserve for extra nonce 2 (which is concatenated with extra nonce 1)
     ESTIMATOR_LOOP_INTERVAL = 30  # in seconds, "frequency" that the function that updates the estimator will be called
     ESTIMATOR_WINDOW_INTERVAL = 60 * 15  # in seconds, size of the window to use for estimating miner's hashrate
-    MIN_DIFFICULTY = 128  # minimum "bitcoin difficulty" to assign to jobs
+    MIN_DIFFICULTY = 50000  # minimum "bitcoin difficulty" to assign to jobs
```

Comment:
It's such a huge increase. Why?

Reply:
Accidentally committed this, I increased here for testing but then noticed I could use the `--min-diff 50000` param.

Reply:
Reversed.